### PR TITLE
Create logo from brandcrowd.com

### PR DESCRIPTION
**Pull request by Jigarvarma2005**

### CODE REVIEW

**Comment by Jigarvarma2005** [comment]
i Can't fix this pep8

**Comment by Phyco-Ninja** [comment]
don't give up just yet

File: plugins/logo_maker.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                            image_link = image["src"]
+                        except:
+                            pass
+            try:
+                r = requests.get(image_link).content
+                try:
+                    r = str(r, "utf-8")
+                except UnicodeDecodeError:
+                    with open(f"{Config.DOWN_PATH}/logo.jpg", "wb+") as f:
+                        f.write(r)
```
Comment (Phyco-Ninja) [inline plugins/logo_maker.py:34]: why is loop still going on when it should exist once file is written? 🤔🤔

File: plugins/logo_maker.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# © JigarVarma2005
+# Logo maker using brandcrowd.com
+# Moded from @JVToolsBot by @UniversalBotsUpdate
+import os
+import random
+import requests
+from bs4 import BeautifulSoup
```
Comment (Phyco-Ninja) [inline plugins/logo_maker.py:7]: could u use aiohttp library instead of requests?

Comment (Jigarvarma2005) [reply plugins/logo_maker.py:7]: Any issue with requests?

Comment (Jigarvarma2005) [reply plugins/logo_maker.py:7]: I will try

**Comment by Krishna-Singhal** [comment]
https://del.dog/licycyllam

copy paste

**Comment by Krishna-Singhal** [comment]
Only line too long left

**Comment by Krishna-Singhal** [comment]
Do it like this 

https://del.dog/tywyjugyga

File: plugins/logo_maker.py [hunk 3]
```diff
@@ -1,67 +1,92 @@
...
     await message.delete()
-    try:
-        os.remove(logo_path)
-    except:
-        pass
+
+
+def main_logo(type_text, type_keyword):
+
+    url = f"https://www.brandcrowd.com/maker/logos?text={type_text}&searchtext={type_keyword}&searchService="
```
Comment (Krishna-Singhal) [inline plugins/logo_maker.py:45]: do it like this 

https://del.dog/tywyjugyga

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: plugins/logo_maker.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    for i in image_list:
+        if os.path.exists(i):
+            await message.client.send_photo(message.chat.id, i)
+            os.remove(i)
+    await message.delete()
+
+
+def main_logo(type_text, type_keyword):
+
+    url = f"https://www.brandcrowd.com/maker/logos?text={type_text}&searchtext={type_keyword}&searchService="
```
Comment (Krishna-Singhal) [inline plugins/logo_maker.py]: just remove `&searchService=` in last

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: plugins/logo_maker.py (hunk 4 above)
Comment (Jigarvarma2005) [reply plugins/logo_maker.py]: Fixed

**Pull request merged**